### DISCUSSION TITLE
Feature Request: Make all Projects (BETA) accessible from primary projects area

### DISCUSSION THREAD

**Discussion by KeygenOld** [community/community#9024] [Projects and Issues]
It's great in the beta you guys added the ability to add an existing project to a repo, however if I create a project inside of a repo, it doesn't show up in my primary Project area accessible from my profile page. There doesn't seem to be a way to add it.

If I've missed something, let me know, but I haven't seen a way to do this. 

All projects we create should be accessible from the main Project section, whether we created them there or inside a repo.

**Comment by Mikibits** [comment]
I agree. Although I'm new to Projects in GitHub, I love the idea of it. Unfortunately there seems to be little documentation of purpose behind there being repo-level projects versus account-level projects. When do you use one over the other? 

The fact that they cannot see each other is also confusing. Perhaps I've just missed a key tutorial on the subject. At the moment it feels like a navigation bugaboo.
